install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2777,6 +2777,9 @@ tls:
 #   - geneve
 # @default -- `"vxlan"`
 tunnelProtocol: ""
+# -- Underlay protocol for the tunnel.
+# @default -- `"ipv4"`
```

Comment:
"protocol" seems a bit overloaded -- one line above, the choices are vxlan or geneve. Here it's ipv4 or ipv6. Should this be called family instead, or do you foresee other, sigh, protocols in use?

Reply:
Are you suggesting changing the flag's name or its description?